tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -160,6 +160,51 @@ annot/Alias.A#
 -              symbol: "scala/annotation/meta/param#"
 
 
+====================
+classes/C1#equals().
+====================
+--- metac
++++ metacp
+   public_access {
+-overriden_symbols: "scala/Any#equals()."
```

Comment:
I'm torn on including overrides for these default methods. I don't think they will be useful for our applications, and they introduce noise in our tests and overhead in the SemanticDB payloads. 

My vote is to include these overrides in the payload because that's the **correct** thing.

Reply:
Should we read the overriden symbols in `scalacp` ? Not sure how to get that information from scalap